cdap-app-fabric/src/main/java/org/apache/hadoop/yarn/util/ApplicationClassLoader.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
No we do not have to keep the duplicate license header because our "derivative" work is under the same ASF 2.0 license.